package.json

```diff
@@ -52,13 +52,11 @@
     "ember-cli-uglify": "^1.2.0",
     "ember-export-application-global": "^1.0.5",
     "ember-gestures": "0.4.2",
-    "ember-getowner-polyfill": "1.1.1",
```

Comment:
Good catch. I got the deprecation that it is in Ember since 2.3. I checked `ember-gestures` and yeah they just have it in the devDependencies. Which is wrong cause the idea of the polyfill is to polyfill for addons if they don't know what version of ember the consuming app uses. I'll readd it just to make sure nothing breaks.